extension/goplugin/util.go

```diff
@@ -0,0 +1,18 @@
+package goplugin
+
+import "github.com/golang/mock/gomock"
+
+var controllers map[gomock.TestReporter]*gomock.Controller
+
+func NewController(testReporter gomock.TestReporter) *gomock.Controller {
+	ctrl := gomock.NewController(testReporter)
+	if controllers == nil {
```

Comment:
this initialization can be moved to declaration so the code here will be cleaner